examples/io/elekta_average.py

```diff
@@ -0,0 +1,56 @@
+"""
+================================
+Getting averaging info from fiff
+================================
+
+Get averaging information defined in Elekta Vectorview/TRIUX DACQ (data
+acquisition). Extract and average epochs accordingly.
+"""
+# Author: Jussi Nurminen (jnu@iki.fi)
+#
+# License: BSD (3-clause)
+
+
+import mne
+import os
+from mne.datasets import testing
+from mne.event import ElektaAverager
+
+elekta_base_dir = os.path.join(testing.data_path(), 'misc')
+fname_raw_elekta = os.path.join(elekta_base_dir, 'test_elekta_3ch_raw.fif')
+
+print(__doc__)
+
+raw = mne.io.read_raw_fif(fname_raw_elekta)
+eav = ElektaAverager(raw.info)
+
+""" Extract epochs corresponding to a category. Copy rejection
+limits from DACQ settings. """
```

Comment:
We usually do comment blocks in examples/tutorials by adding a line of #s and then one hashtag for each line. This way it renders nicely on the website and makes nice blocks on notebooks (I suppose).
Also FYI, you can make it automatically run for the website by starting the file name with ``plot_``. That's nice if you have something to plot. Since you don't plot anything here, I think it's okay like this.